objects/src/accounts/testing/chain.rs

```diff
@@ -26,7 +32,6 @@ const TIMESTAMP_START: u32 = 1693348223;
 const TIMESTAMP_STEP: u32 = 10;
 
 #[derive(Default, Debug, Clone)]
-#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
```

Comment:
why was this derive removed?

Reply:
Not needed/used anywhere. `MockChain` could probably be removed altogether but I was planning to reuse some of this in the next PRs.